bbn-test-4/finality-providers/README.md

```diff
@@ -0,0 +1,214 @@
+# Finality Provider Information Registry
+
+Bitcoin holders that stake their Bitcoin have the option to delegate their
+attestation of power to finality providers by including the finality provider
+public key in the self-custodial Bitcoin Staking script.
+While the finality provider key is the only identifying information required
+for a finality provider, it does not expose all the information that a
+finality provider might want to share to attract more stake delegations.
+
+In this guide, we describe how finality providers wishing to participate in the
+Babylon lock-only testnet that connects to the Bitcoin signet can register
+their information. The aim is to allow finality providers to share their identifying
+information in a way that proves their ownership of the finality provider.
+Finality providers can pre-register either before the testnet launch, or
+provide information after it. *Note, that this registry only serves an
+informational purpose: the only way for a finality provider to register
```

Comment:
what does "register for information purpose" and "register protocol wise" mean?

Let's make this instruction more clear with the following flow:

1. Babylon's web dapp will display a list of FPs.
2. Any EOTS PK that receives delegations will be listed there.
3. But for human-readability, we will also display more information, such as name, website, etc.
4. Thus, we create this repo for FPs to register such info via submitting PRs to it.
5. To prevent spam, we request FPs to deposit sBTC and include the proof as part of the PR.
6. You can do it before or after launch. But to be included in the initial list in our web dapp, please do it before launch.

Here is how: